app/helpers/application_helper.rb

```diff
@@ -291,4 +291,8 @@ def admin_config_label(method, content = nil)
     tag.label(content, class: "site-config__label crayons-field__label", for: "site_config_#{method}")
   end
   # rubocop:enable Rails/OutputSafety
+
+  def admin_config_description(content)
+    tag.p(content, class: "crayons-field__description")
```

Comment:
I now understand where that [`&:empty { display: none; }`](https://github.com/forem/forem/pull/10874/files#r506547366) came from in CSS.. :). Instead making that CSS change, I'd suggest doing the following thing:

```suggestion
    tag.p(content, class: "crayons-field__description") unless content.empty?
```

It will prevent generating code for description if it's empty. BUT please check with some experienced developer if this is correct (it does work on my end though so should be good :P).

Reply:
@joshpuetz Is this recommendation that @ludwiczakpawel is making okay for us?

Reply:
@ludwiczakpawel Your solution works well for me too :)